notebook/run-jupyter.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
For ease of deployment on a new IDAaaS workspace could we make this a `curl ... | sh -`able script that will also take care of downloading the notebook.

Reply:
We should catch up about the details about this